### PR TITLE
[NP-961] ModelSectionData

### PR DESCRIPTION
**Pull request by KernelDeimos**
This PR updates CrunchController and ScrollSectionWizardView, removing anonymous objects and making it possible to use ScrollSectionWizardView for CRUNCH-unrelated wizards.

### CODE REVIEW

**Comment by Nauna** [comment]
- remove pseudo comments

File: src/foam/nanos/crunch/ui/CapabilityWizardSection.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      name: 'daoFindKey',
+      expression: function (capability) {
+        return capability.daoFindKey
+      }
+    },
+    {
+      name: 'ofSections',
+      factory: null,
+      expression: function(of) {
+        console.log('of', of);
```
Comment (KernelDeimos) [inline src/foam/nanos/crunch/ui/CapabilityWizardSection.js:55]:
```suggestion
```

File: src/foam/nanos/crunch/ui/CapabilityWizardSection.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        return ucj.data;
+      }
+    }
+  ],
+
+  actions: [
+    {
+      name: 'save',
+      code: function() {
+        console.log('action this', this);
```
Comment (KernelDeimos) [inline src/foam/nanos/crunch/ui/CapabilityWizardSection.js:81]:
```suggestion
```

File: src/foam/nanos/crunch/ui/ScrollSectionWizardView.js [hunk 3]
```diff
@@ -50,8 +50,9 @@ foam.CLASS({
       name: 'isErrorFree',
       expression: function(sectionsList) {
         var check = true;
-        sectionsList.forEach((m) => {
-          if ( m.data.errors_ ) {
+        sectionsList.forEach((wizardSection) => {
+          console.log(wizardSection.data)
```
Comment (KernelDeimos) [inline src/foam/nanos/crunch/ui/ScrollSectionWizardView.js:54]:
```suggestion
```

**Pull request merged**